### DISCUSSION TITLE
Welcome to Hacktoberfest2k21 Discussions!

### DISCUSSION THREAD

**Discussion by fineanmol** [fineanmol/Hacktoberfest2021#274] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sadeghhosseiny** [comment]
> ## 👋 Welcome!
> We’re using Discussions as a place to connect with other members of our community. We hope that you:
> 
> * Ask questions you’re wondering about.
> * Share ideas.
> * Engage with other community members.
> * Welcome others and are open-minded. Remember that this is a community we
>   build together 💪.
> 
> To get started, comment below with an introduction of yourself and tell us about what you do with this community.

dude, merge my branch too, please

**Reply by fineanmol** [reply]
Please share your Pull request Number so I can review and merge it.

**Reply by sadeghhosseiny** [reply]
where's the pull request number, where can i find it??

**Reply by sadeghhosseiny** [reply]
![image](https://user-images.githubusercontent.com/56449965/147249779-3aab826d-56ce-4ede-a796-a44aa889bf62.png)

**Reply by fineanmol** [reply]
You have to click on Create pull request, only after that I'll receive your pull request.

**Reply by sadeghhosseiny** [reply]
dude i ever contribute to any project untill i saw yorus. please help me to do that. i really love to contribute to your project beacause this is a 
large community and i read some articles about pull request and i did it like them.  just say what should i do.
when i  made a new branch and commit that, i saw a merge branches button wich is not in the movies that i saw. they said the author can only merge it. and i don't know i did it wrong or not. so what should i do ?

**Reply by sadeghhosseiny** [reply]
> You have to click on Create pull request, only after that I'll receive your pull request.

dude, this time i pulled request correctly :))
merge it, please
